typescript/infra/helm/warp-routes/templates/_helpers.tpl

```diff
@@ -72,11 +72,11 @@ The warp-routes container
      value: pretty
   command:
   - ./node_modules/.bin/tsx
-  - ./typescript/infra/scripts/warp-routes/monitor-warp-routes-balances.ts
+  - ./typescript/infra/scripts/warp-routes/monitor/monitor-warp-routes-balances.ts
```

Comment:
this is the most minor of nits ever but should the script instead be 
```diff
-monitor-warp-routes-balances.ts
+monitor-warp-route-balances.ts
```

Reply:
lol yeah ill rename